src/main/java/org/spongepowered/common/mixin/core/entity/MixinEntity.java

```diff
@@ -774,6 +776,34 @@ public DataTransactionResult addPassenger(Entity entity) {
     }
 
     @Override
+    public DataTransactionResult removePassenger(Entity entity) {
+        checkNotNull(entity);
+        if (entity.getPassengers().contains(this)) {
+            throw new IllegalArgumentException(String.format("Cannot remove entity %s as a passenger of %s, because the former is not a passenger "
+                    + "of the latter ", entity, this));
+        }
+
+        final DataTransactionResult.Builder builder = DataTransactionResult.builder();
+        this.removePassenger((net.minecraft.entity.Entity) entity);
+
+        final ImmutableList.Builder<EntitySnapshot> passengerSnapshotsBuilder = ImmutableList.builder();
```

Comment:
I accidentally moved that one as well as clearPassengers, so not intended to be empty :P